src/mantine-dates/src/components/TimeInput/TimeInput.tsx

```diff
@@ -35,17 +40,38 @@ export const TimeInput = factory<TimeInputFactory>((_props, ref) => {
     props,
   });
 
+  const validTime = () => {
```

Comment:
This function must be outside of component body, put it in the separate file

Reply:
Created new file for this function